draft-ietf-ppm-dap.md

```diff
@@ -1483,20 +1493,34 @@ enc, payload = SealBase(pk,
 * `server_role` is the `Role` of the recipient (`0x02` for the Leader and `0x03`
    for the Helper).
 * `plaintext_input_share` is the Aggregator's `PlaintextInputShare`.
-* `input_share_aad` is an encoded `InputShareAad`, constructed from the
-  corresponding fields in the report per the definition below.
+* `input_share_aad` is an encoded `InputShareAad`, defined below.
 
 The `SealBase()` function is as specified in {{!HPKE, Section 6.1}} for the
 ciphersuite indicated by the Aggregator's HPKE configuration.
 
 ~~~ tls-presentation
+uint32 VdafIdentifier;
+
 struct {
   TaskID task_id;
   ReportMetadata report_metadata;
   opaque public_share<0..2^32-1>;
+  VdafIdentifier vdaf_identifier;
```

Comment:
I went back and forth on whether we need `vdaf_identifier`. `InputShareAad` never gets transmitted or decoded, it's only constructed by clients to encrypt messages and then constructed by aggregators to decrypt them. So you don't need an enum discriminant to tell you how to decode `vdaf_configuration`. However, it's possible that the configurations for two different VDAF types could have the same encoding, so I believe we need the VDAF codepoint to reliably distinguish them.